frontend/src/metabase/lib/urls/questions.ts

```diff
@@ -22,7 +22,9 @@ export type QuestionUrlBuilderParams = {
 };
 
 export function question(
-  card: Card | null,
+  card: Partial<
+    Pick<Card, "id" | "name" | "type" | "card_id" | "model">
+  > | null,
```

Comment:
I might be off my head but would `Partial<Card> | null` work here?

Reply:
It would work, but I kind of like how `Pick` describes what the component actually might use. WDYT?

Reply:
Sure, I dig it!